Dockerfile

```diff
@@ -1,6 +1,7 @@
 FROM rust:1.41.0 AS builder
 COPY . lighthouse
 RUN cd lighthouse && make && cargo clean
```

Comment:
I think the `cargo clean` should be moved after the build of `lcli` so that the second build can use the shared crates?

Reply:
Nice one! I'll remove the `cargo clean` completely since the builder container gets dropped anyway :)